### DISCUSSION TITLE
Need help adding style attributes like width and height to the table element while editing style properties using Style Manager

### DISCUSSION THREAD

**Discussion by itsmesridhar** [GrapesJS/grapesjs#4710] [Q&A] [answered]
With the help of **Grapesjs** we are developing a newsletter designer with HTML tables instead of divs. Even though outlook client doesn't support CSS styles for table elements.  Since it doesn't support CSS style we plan to add width and height as attributes. But currently, while modifying the width and height properties in the style manager it's appending as a CSS style instead of direct width and height attributes for HTML tables. Need some solution to achieve this. Thanks!
```
<table style="width: 100%; height: 25px"></table>
```
Expected:
```
<table style="width: 100%; height: 25px" width="100%" height="25"></table>
```

**Comment by artf** [comment, marked as answer]
I'd suggest following what the [grapesjs-preset-newsletter is doing](https://github.com/artf/grapesjs-preset-newsletter/blob/master/src/openExportCommand.js) with the export code by relying on something like [juice](https://www.npmjs.com/package/juice) to inline styles (by default juice is also handling attributes for the tables)
